analysis/pseudobulk-bulk-prediction/model-notebooks/utils.R

```diff
@@ -0,0 +1,47 @@
+# This script holds helper functions used by notebooks in the `model-notebooks` directory
+
+
+#' Prepare data for analysis
+#'
+#' This function reads and combines bulk and pseudobulk data into a single
+#' data frame for a given project
+#'
+#' @param bulk_counts_file Path to RDS file with bulk counts for all projects
+#' @param pseudobulk_file Path to pseudobulk TSV file for the project of interest
+#' @param project_id Project id of interest, used to filter the bulk_counts
+#' @param exclude_samples Optional vector of samples to remove from the final data frame
+#'
+#' @returns
+#' @export
+#'
+#' @examples
+prepare_data <- function(
+    bulk_counts_file,
+    pseudobulk_file,
+    project_id,
```

Comment:
If you are going to map across projects (which I think you should) it may be easier with `project_id` first.
```suggestion
prepare_data <- function(
    project_id,
    bulk_counts_file,
    pseudobulk_file,
```